.buildkite/pull-requests.json

```diff
@@ -2,11 +2,10 @@
   "jobs": [
     {
       "enabled": true,
-      "pipelineSlug": "logstash-filter-elastic_integration-pull-request-pipeline",
+      "pipelineSlug": "logstash-filter-elastic_integration-pull-request",
```

Comment:
to avoid future confusion, we could also rename https://github.com/elastic/logstash-filter-elastic_integration/blob/a71a0700168724e089dd247f01ceb799f082af66/catalog-info.yaml#L39 and https://github.com/elastic/logstash-filter-elastic_integration/blob/a71a0700168724e089dd247f01ceb799f082af66/catalog-info.yaml#L43 to point to the same string: `logstash-filter-elastic_integration-pull-request-pipeline`

Reply:
Yup, thank you! I have updated them as well.